src/main/scala/lms/thirdparty/cuda/cuda.scala

```diff
@@ -429,10 +431,18 @@ trait CudaOps extends Dsl with StackArrayOps with SizeTOps with CLibs with CudaF
       val offset = skip * i + j
       Wrap[T](Adapter.g.reflectRead("array_get", Unwrap(x), Unwrap(offset))(Unwrap(x)))
     }
+    def apply(is: List[Rep[Int]])(implicit __pos: SourceContext): Rep[T] = {
```

Comment:
what is the benefit for using `List[Rep[Int]]` instead of using 2 parameters?

Reply:
just convenient if the indices are already in a list :)